source/crud/configure.txt

```diff
@@ -20,81 +20,225 @@ Configure CRUD Operations
 Overview
 --------
 
-In this guide, you can learn how to configure read and write operations in {+driver-short+}.
+In this guide, you can learn how to use the {+driver-short+} to configure read
+and write operations.
 
-.. _node-fundamentals-collations:
-.. _node-collations:
+Read and Write Settings
+-----------------------
 
-Collations
-----------
+You can control how the driver routes read operations by setting a **read preference**.
+You can also control options for how the driver waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
```

Comment:
```suggestion
You can also control options for how the driver waits for acknowledgment of write operations on a replica set by setting a **write concern** and what level of durability the server has achieved for the data being queried when reading data by setting a **read concern**.
```

slight nit: the existing phrasing reads to me to imply that read concern impacts write durability, which isn't accurate

Reply:
I reworded this section, let me know what you think!